### PR TITLE
Update README.md with wasmcert

### PR DESCRIPTION
**Pull request by spitters**

### CODE REVIEW

**Comment by spitters** [comment]
Build failed because @JasonGross 's category theory library seems to have moved...
https://plv.csail.mit.edu/blog/category/coq.html

**Comment by palmskog** [comment]
@spitters I'm hesitant to add this for several reasons:
- the GitHub repo says this formalization has not been validated against WASM conformance tests
- not clear if it will be maintained going forward
- no mention on any research publication for this specific formalization

Can you comment on this and the general status of this work?

**Comment by JasonGross** [comment]
> Build failed because @JasonGross 's category theory library seems to have moved... https://plv.csail.mit.edu/blog/category/coq.html

That's not my category theory library, but the posts on the plv blog in the category "Coq".  And the problem is the https, http://plv.csail.mit.edu/blog/category/coq.html seems to work fine

**Comment by spitters** [comment]
We're using the library for the wasm backend of certicoq. I'll let the maintainers of wasm-cert answer the other questions...

**Comment by spitters** [comment]
@raoxiaojia Maybe you can update the README to distinguish TODOs from future work.
It would also be nice to add the publication, so you get credit for your work.

@palmskog with these changes would you be happy?

**Comment by raoxiaojia** [comment]
> @raoxiaojia Maybe you can update the README to distinguish TODOs from future work. It would also be nice to add the publication, so you get credit for your work.
> 
> @palmskog with these changes would you be happy?

Readme is now updated to reflect my reply above, adding a bit more details on the separation between the works done at original publication and those since then. I think this is indeed helpful to the repo as you stated, and provides more information in general.

File: README.md [hunk 1]
```diff
@@ -201,6 +201,7 @@ Contributions welcome! Read the [contribution guidelines](https://github.com/coq
 - [Tarjan and Kosaraju](https://github.com/math-comp/tarjan) - Verified implementations of algorithms for topological sorting and finding strongly connected components in finite graphs.
 - [Vélus](http://velus.inria.fr) - Verified compiler for a Lustre/Scade-like dataflow synchronous language.
 - [Verdi Raft](https://github.com/uwplse/verdi-raft) - Implementation of the Raft distributed consensus protocol, verified in Coq using the Verdi framework.
+- [WasmCert-Coq](https://github.com/WasmCert/WasmCert-Coq/) - WebAssembly (aka Wasm) 1.0 formalisation in Coq
```
Comment (palmskog) [inline README.md:204]:
```suggestion
- [WasmCert-Coq](https://github.com/WasmCert/WasmCert-Coq/) - Formalization in Coq of the WebAssembly (aka Wasm) 1.0 specification.
```

**Review by palmskog — COMMENTED** [review]
I think the explanation here and the current project README looks good enough for including this. However, can we do a slight change in the list entry here @spitters?

**Comment by spitters** [comment]
@palmskog What would you like me to do with these:
  1. [L235]  http://coq-blog.clarus.me Failed to open TCP connection to coq-blog.clarus.me:80 (Connection refused - connect(2) for "coq-blog.clarus.me" port 80) 
  2. [L272]  https://www.ssi.gouv.fr/uploads/2014/11/anssi-requirements-on-the-use-of-coq-in-the-context-of-common-criteria-evaluations-v1.1-en.pdf SSL_connect returned=1 errno=0 state=error: certificate verify failed (unable to get local issuer certificate)

**Comment by palmskog** [comment]
@spitters unfortunately the `clarus.me` blog seems to be permanently down, I'd appreciate if you removed it.

The `ssi.gouv.fr` link still works, it's just that some certificate expired, and they disallow non-HTTPS connections. I think we'll have to keep that one as is.

**Comment by spitters** [comment]
@palmskog I removed the blog.

**Pull request merged**